constellations/orion/src/tests_integration/family_integration_test.go

```diff
@@ -0,0 +1,178 @@
+package integration_tests
+
+import (
+	"bytes"
+	"encoding/json"
+	"net/http"
+	"testing"
+
+	"github.com/ahsu1230/mathnavigatorSite/constellations/orion/src/domains"
+	"github.com/stretchr/testify/assert"
+)
+
+type FamilySearchBody struct {
```

Comment:
Looks like we already defined this type in the controllers...
Maybe we can create a search body another way.